test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -417,4 +420,395 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
     assert.equal(createdTrustLine.limit, trustLineLimit)
     assert.equal(createdTrustLine.currency, trustLineCurrency)
   })
+
+  // TODO: (acorso) can any addresses be created in `before` and reused?  Can any test cases be combined?
+  it('sendIssuedCurrency - issuing issued currency, combined cases', async function (): Promise<
```

Comment:
this function is extremely core to `issuedCurrencyClient` is half the code in this test - could we put this in its own file? `issued-currency-client-sendissuedcurrency` or something?

Reply:
yeah... I feel like all of these tests are going to change very soon when I instead implement specific, opinionated APIs / I made that Asana task for a full audit of these issued currency client tests... lemme think about this for a second.

Reply:
nah this is cool for now. let's roll with this and figure out a better plan for how we organize tests - this is a grander problem in the sdk